py/client/pydeephaven/session.py

```diff
@@ -127,12 +138,37 @@ def get_ticket(self):
 
             return self._last_ticket
 
+    def subscribe_fields(self):
+        """
+        Allows tables created/removed on the server to be automatically seen on the client
+        """
```

Comment:
1. should we make it private? If so it could save us a lot of documentation on how it should be typically used it and the implication of the dynamically changing nature of the fields, until such need is requirement.
2. the docstring format should be """ Allows ... """ to be consistent with the rest of the code.

Reply:
IMO, I think we do want a public method that is opt-in (or have a boolean on connect).

Reply:
This is now available as `Session(sync_fields=SYNC_ONCE)` or as `session.sync_fields(repeated=False)`